src/main/java/tudelft/domain/TwoIntegers.java

```diff
@@ -0,0 +1,11 @@
+package tudelft.domain;
+
+public class TwoIntegers {
+
+  public int sum(int n, int m) {
+    assert n >= 1 && n <= 99;

```

Comment:
assertions are just beautiful, aren't they? :)

Reply:
they keep the code clean as hell 😄